OpenOversight/app/commands.py

```diff
@@ -227,6 +227,35 @@ def create_officer_from_row(row, department_id):
     process_salary(row, officer, compare=False)
 
 
```

Comment:
This was added - there was a weird issue of failing unit tests on the first PR. I updated the test data to have datetime's for start_date + resign_date on assignments. Without this equality check, we would be == comparing datetime(2020, 01, 01) to '2020-01-01', which would fail and we would trigger spurious row updates and fail the tests.